apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,13 +1,13 @@
 ack_generate_info:
-  build_date: "2023-01-10T21:58:28Z"
-  build_hash: 1b20baf45a0b73a11b296050322a384c705fa897
-  go_version: go1.17.5
-  version: v0.22.0
-api_directory_checksum: ee32acc4d4a0ba7e2823dd20fdbe2c4ef1d9e0f4
+  build_date: "2023-01-19T23:23:40Z"
+  build_hash: c6651c200ba136e5c7f50ad8be751fae060a38e6
+  go_version: go1.19
+  version: v0.21.0-9-gc6651c2
```

Comment:
nit: You will need to `git fetch --tags` in your `code-generator` repository, because it doesn't know about the latest version

Reply:
Just did it and updated it~